larva/src/main/java/nl/nn/adapterframework/testtool/TestTool.java

```diff
@@ -4203,138 +4205,123 @@ public static String replaceRegularExpression(String string, String from, String
 	 */
 	private static Map<String, Object> createParametersMapFromParamProperties(Properties properties, String property, Map<String, Object> writers, boolean createParameterObjects, PipeLineSession session) {
 		debugMessage("Search parameters for property '" + property + "'", writers);
-		Map<String, Object> result = new HashMap<String, Object>();
+		final String _name = ".name";
+		final String _param = ".param";
+		final String _type = ".type";
```

Comment:
I hate to say it but this does make it more difficult to figure out what has actually been changed :)

Reply:
yes, but later on it helps to find out where it is used!